R/ddi.R

```diff
@@ -9,12 +15,23 @@
 #' \dontrun{
 #' rifampicin <- compound("Rifampicin")
 #' midazolam <- compound("Midazolam")
-#' create_ddi(rifampicin, midazolam)
+#' create_ddi(victim = rifampicin, perpetrator = midazolam)
 #' }
-create_ddi <- function(...) {
+create_ddi <- function(victim, perpetrator) {
+  if (missing(victim)) {
+    cli::cli_abort("At least one victim compound must be provided.")
+  }
+  if (missing(perpetrator)) {
+    cli::cli_abort("At least one perpetrator compound must be provided.")
+  }
+
+  victim <- to_list(victim)
```

Comment:
Why do we convert to list ? victim should always be of length 1.

Reply:
Yes, we don't really need it.